otter/integration/tests/test_lbsh.py

```diff
@@ -599,3 +602,54 @@ def test_changing_disowned_server_is_not_converged_2(self):
                 timeout=timeout_default
             )
         ])
+
+    @inlineCallbacks
+    def test_draining(self):
```

Comment:
I've tested this against prod. I am _hoping_ that `30 + 5 + convergence_interval * 2` should make it work in prod as long as `convergence_interval` matches one in prod.